tests/tests_unit/rules/test_models/test_asset_rules.py

```diff
@@ -0,0 +1,362 @@
+        assert polygon is not None
+        assert polygon.implements == [dm.ViewId("power", "Polygon", "0.1.0")]
+
+
+class TestAssetRulesConverter:
+    @pytest.mark.parametrize(
+        "prefix, expected_space",
+        [
+            pytest.param("neat", "neat", id="No substitutions"),
+            pytest.param("my space", "my_space", id="Space with space character"),
+            pytest.param("1my-space", "a1my-space", id="Space starting with number"),
+            pytest.param("m" * 69, "m" * 43, id="Space with more than 43 characters"),
+            pytest.param("my_space_", "my_space1", id="Space ending with underscore"),
+        ],
+    )
+    def test_to_space(self, prefix: str, expected_space: str) -> None:
+        actual_space = _AssetRulesConverter._to_space(prefix)
+
+        assert actual_space == expected_space
+
+    def test_svein_harald_information_as_dms(self, svein_harald_information_rules: AssetRules) -> None:
+        expected = {
+            "ArrayCable": {"PowerLine"},
+            "DistributionLine": {"PowerLine"},
+            "DistributionSubstation": {"Substation"},
+            "ElectricCarCharger": {"EnergyConsumer"},
+            "ExportCable": {"PowerLine"},
+            "MultiLineString": {"GeoLocation"},
+            "OffshoreSubstation": {"Substation"},
+            "OnshoreSubstation": {"TransmissionSubstation"},
+            "Point": {"GeoLocation"},
+            "Polygon": {"GeoLocation"},
+            "Transmission": {"PowerLine"},
+            "TransmissionSubstation": {"Substation"},
+            "WindFarm": {"EnergyArea"},
+            "WindTurbine": {"GeneratingUnit"},
+        }
+
+        dms_rules = svein_harald_information_rules.as_dms_architect_rules()
+
+        assert isinstance(dms_rules, DMSRules)
+        assert dms_rules.last is not None
+        actual = {
+            view.view.external_id: {parent.external_id for parent in view.implements}
+            for view in dms_rules.last.views
+            if view.implements
+        }
+
+        assert actual == expected
+
+    def test_convert_above_container_limit(self) -> None:
+        info = AssetRulesInput(
+            metadata=AssetMetadataInput(
+                schema_="complete",
+                prefix="bad_model",
+                namespace="http://purl.org/cognite/bad_model",
+                name="Bad Model",
+                version="0.1.0",
+                creator="Anders",
+            ),
+            classes=[AssetClassInput(class_="MassiveClass")],
+            properties=[
+                AssetPropertyInput(
+                    class_="MassiveClass",
+                    property_=f"property_{no}",
+                    value_type="string",
+                )
+                for no in range(DMS_CONTAINER_SIZE_LIMIT + 1)
+            ],
+        ).as_rules()
+
+        dms_rules = info.as_dms_architect_rules()
+
+        assert len(dms_rules.containers) == 2
```

Comment:
Since the AssetConverter is inheriting from the Information Converter with no modifications, I don't think this is necessary to test.
```suggestion
```